### DISCUSSION TITLE
Some advises about 2FA in China

### DISCUSSION THREAD

**Discussion by COMPUTER102** [community/community#67597] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In China, it's not easy to install 2FA software: **LOTS OF** 2FA software have to download on Google Play, but we **CAN'T** use Google Play in China! (Some of the 2FA software also can download on App Store, but in China, it's **NOT EASY** to install either), but GitHub requires all users to use 2FA.

Ther are many Chinese users on GitHub. This will cause us a **huge** impact and **a lot of** trouble.

**Comment by jzelAdmin2006** [comment]
Hi @COMPUTER102

I understand the challenges you face with 2FA in China. My recommendation is to explore other sources for downloading TOTP software. One that I personally vouch for is Authy. It's available for PC, Linux, or Mac and can be downloaded here: https://authy.com/download/. If you're keen on having it on your smartphone, you can use a site like APKMirror to download the APK and install it without Google Play on an Android smartphone. Here's the link: https://www.apkmirror.com/?post_type=app_release&searchtype=apk&s=authy.

I hope this helps!

Best regards
jzelAdmin2006

**Comment by someview** [comment]
> Hi @COMPUTER102
> 
> I understand the challenges you face with 2FA in China. My recommendation is to explore other sources for downloading TOTP software. One that I personally vouch for is Authy. It's available for PC, Linux, or Mac and can be downloaded here: https://authy.com/download/. If you're keen on having it on your smartphone, you can use a site like APKMirror to download the APK and install it without Google Play on an Android smartphone. Here's the link: https://www.apkmirror.com/?post_type=app_release&searchtype=apk&s=authy.
> 
> I hope this helps!
> 
> Best regards jzelAdmin2006

Why not think about support SMS code in china，It‘s easy to do.
